database/index.js

```diff
@@ -1,3 +1,28 @@
+require('dotenv').config();
+const PORT = 8529;
+const { Database, aql } = require('arangojs');
```

Comment:
do you use aql here?

Reply:
I was initially intending to. However, in the end I did not. I'll remove that.